src/components/Select/types.d.ts

```diff
@@ -12,5 +14,6 @@ export interface SelectProps {
 export interface SelectOption {
   group: string,
   label: string,
-  value: any,
+  role: Role,
```

Comment:
Role should not be here. If it's needed as a type for a specific use extend this type in the place it's needed.